supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesPollingActor.scala

```diff
@@ -107,13 +108,10 @@ object JesPollingActor {
     * Given the Genomics API queries per 100 seconds and given MaxBatchSize will determine a batch interval which
     * is at 90% of the quota. The (still crude) delta is to provide some room at the edges for things like new
     * calls, etc.
-    *
-    * Forcing the minimum value to be 1 second, for now it seems unlikely to matter and it makes testing a bit
-    * easier
     */
   def determineBatchInterval(qps: Int Refined Positive): FiniteDuration = {
-    val maxInterval = MaxBatchSize / qps.value
-    val interval = Math.max(maxInterval / 0.9, 1).toInt
-    interval.seconds
+    val maxInterval = MaxBatchSize.toDouble / qps.value.toDouble
+    val interval = ((maxInterval / 0.9) * 1000).toInt
+    interval.milliseconds
   }
```

Comment:
From the review comments, three orders of magnitude seems like kind of a big change in the frequency limit all at once. 😬

Reply:
TOL even if we stick with the `Refined Positive` qps after my earlier gulling, it's now less clear here that the result of this expression is constrained to be positive.

Reply:
About your first comment I don't think it matters too much, because this time value is actually a delay we wait for before doing the next request, and after we finish our current one. So we won't go faster than we can anyway, we might go as fast as we can + 1ms instead of +1s.